source/instructors/admin/integration/lti1-3Canvas.rst

```diff
@@ -6,151 +6,286 @@
+|     </div>                                                                              |                                                                                                                                                              |
++-----------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| .. raw:: html                                                                           |                                                                                                                                                              |
+|                                                                                         |                                                                                                                                                              |
+|     <div class="small">                                                                 |                                                                                                                                                              |
+|                                                                                         |    |image5|                                                                                                                                                  |
+|     12. Expand the <strong>LTI Advantage Services</strong> section and toggle           |                                                                                                                                                              |
+|         each field on.<br><br>                                                          |                                                                                                                                                              |
+|     13. Expand the <strong>Additional Settings</strong> section<br><br>                 |                                                                                                                                                              |
+|                                                                                         |                                                                                                                                                              |
+|     14. Type "codio.com" in both the <strong>Domain</strong> and                        |                                                                                                                                                              |
```

Comment:
The numbering for points 14, 15 and 16 don't match what I see on the graphics when I build it:

![image](https://github.com/user-attachments/assets/b56abeaf-be8e-4569-b98b-e400260add6c)